src/lavinmq/http/controller/connections.cr

```diff
@@ -17,6 +17,12 @@ module LavinMQ
     class ConnectionsController < Controller
       include ConnectionsHelper
 
+      protected def match_value(value)
+        if client_properties = (value[:client_properties]? || value["client_properties"]?)
```

Comment:
so if there are not client_properties it's not searchable at all?

Reply:
Yeah, but client_properties should always exist as far as I can tell.

Reply:
maybe not for mqtt connections..